process_report/config.py

```diff
@@ -0,0 +1,175 @@
+PREPAY_CONTACTS_FILEPATH = "prepaid_contacts.csv"
+
+# Defaults to being fetched from S3
+CSV_INVOICE_FILEPATH_LIST = []
+PREPAY_DEBITS_FILEPATH = ""
+OLD_PI_FILEPATH = ""
+ALIAS_FILEPATH = ""
+
+# Defaults to being fetched in ColdfrontFetchProcessor
+COLDFRONT_API_FILEPATH = ""
+
+
+### Miscellaneous config values TODO: Should these have their own getter functions?
+NEW_PI_CREDIT_AMOUNT = rates_info.get_value_at("New PI Credit", INVOICE_MONTH, Decimal)
+LIMIT_NEW_PI_CREDIT_TO_PARTNERS = rates_info.get_value_at(
+    "Limit New PI Credit to MGHPCC Partners", INVOICE_MONTH, bool
+)
+BU_SUBSIDY_AMOUNT = rates_info.get_value_at("BU Subsidy", INVOICE_MONTH, Decimal)
+
+
+### Getter functions. Don't change these
+@functools.lru_cache
+def get_csv_invoice_filepaths() -> list[str]:
+    if not CSV_INVOICE_FILEPATH_LIST:
+        return fetch_s3_invoices(INVOICE_MONTH)
+
+
+@functools.lru_cache
+def get_prepay_debits_filepath() -> str:
+    if not PREPAY_DEBITS_FILEPATH:
+        return util.fetch_s3(PREPAY_DEBITS_S3_FILEPATH)
+
+
+@functools.lru_cache
+def get_old_pi_filepath() -> str:
+    if not OLD_PI_FILEPATH:
+        return util.fetch_s3(PI_S3_FILEPATH)
+
+
+@functools.lru_cache
+def get_alias_filepath() -> str:
+    if not ALIAS_FILEPATH:
+        return util.fetch_s3(ALIAS_S3_FILEPATH)
+
+
+@functools.lru_cache
+def get_nonbillable_pis() -> list[str]:
+    with open(NONBILLABLE_PIS_FILEPATH) as file:
+        return [line.rstrip() for line in file]
+
+
+@functools.lru_cache
+def get_nonbillable_projects() -> list[str]:
+    """Returns list of nonbillable projects for current invoice month"""
+    with open(NONBILLABLE_PROJECTS_FILEPATH) as file:
+        projects = [line.rstrip() for line in file]
+
+    timed_projects_list = get_nonbillable_timed_projects(
+        NONBILLABLE_TIMED_PROECTS_FILEPATH, INVOICE_MONTH
+    )
+    return list(set(projects + timed_projects_list))
+
+
+@functools.lru_cache
+def get_nonbillable_timed_projects(timed_projects_file, invoice_date) -> list[str]:
+    """Returns list of projects that should be excluded based on dates"""
+    dataframe = pandas.read_csv(timed_projects_file)
+
+    # convert to pandas timestamp objects
+    dataframe["Start Date"] = pandas.to_datetime(
+        dataframe["Start Date"], format="%Y-%m"
+    )
+    dataframe["End Date"] = pandas.to_datetime(dataframe["End Date"], format="%Y-%m")
+
+    mask = (dataframe["Start Date"] <= invoice_date) & (
+        invoice_date <= dataframe["End Date"]
+    )
+    return dataframe[mask]["Project"].to_list()
+
+
+@functools.lru_cache
+def get_alias_map() -> dict:
+    alias_dict = dict()
+    with open(ALIAS_FILEPATH) as f:
+        for line in f:
+            pi_alias_info = line.strip().split(",")
+            alias_dict[pi_alias_info[0]] = pi_alias_info[1:]
+
+    return alias_dict
+
+
+@functools.lru_cache
+def get_prepaid_credits_df() -> pandas.DataFrame:
+    pandas.read_csv(PREPAY_CREDITS_FILEPATH)
```

Comment:
This function does not return anything. All of the tests are passing, which suggests that this function isn't tested. In fact, a test coverage report shows that most of the functions in this file are not tested.